### PR TITLE
Update to ask for approval

### PR DESCRIPTION
**Pull request by joel99**
Sorry for the gross merge

### CODE REVIEW

File: scripts/notifications.coffee [hunk 1]
```diff
@@ -40,7 +39,17 @@ GRAFANA = {
   tags: {}
 }
 
-PRESETS = ['hackers', 'mediums', 'help']
+PRESETS = ['buildgt', 'mediums', 'help', 'approve', 'abort']
```
Comment (ehsanmasdar) [inline scripts/notifications.coffee]: Looks like a bit of a merging issue, we want the `hackers` preset (buildgt is old).

File: scripts/notifications.coffee [hunk 2]
```diff
@@ -213,4 +252,30 @@ e.g. --message Hello World --channel general tech --> "Hello World" used as --me
       # Add necessary configurations to vars
       vars.plugins[medium] = varsTemp[medium]
 
-    doRequest(vars, res)
+    # Verify message:
+    robot.messageRoom CONTROL_CHANNEL_NAME, "New notification pending. How's it look? (type `botgt notify approve` or `botgt notify abort`)"
```
Comment (ehsanmasdar) [inline scripts/notifications.coffee]: Can we check if there's a already pending message (by my understanding, this code would overwrite the previous pending message with the new pending message)

File: scripts/notifications.coffee [hunk 3]
```diff
@@ -213,4 +252,30 @@ e.g. --message Hello World --channel general tech --> "Hello World" used as --me
...
+    robot.messageRoom CONTROL_CHANNEL_NAME, configToLog(logDict)
+    robot.brain.set('pending', JSON.stringify(vars))
+    robot.brain.set('sender', res.envelope.user.name)
+    # robot.messageRoom VERIFY_CHANNEL_NAME, {
```
Comment (ehsanmasdar) [inline scripts/notifications.coffee]: Remove the commented block

File: scripts/notifications.coffee [hunk 4]
```diff
@@ -213,4 +252,30 @@ e.g. --message Hello World --channel general tech --> "Hello World" used as --me
...
+    #       {
+    #           "name": "choice",
+    #           "text": "Abort",
+    #           "type": "button",
+    #           "value": "cancel"
+    #       }
+    #     ]
+    #   }]
+    # }
+    res.send ''
```
Comment (ehsanmasdar) [inline scripts/notifications.coffee]: Why res.send nothing here?

**Review by ehsanmasdar — CHANGES_REQUESTED** [review]

File: scripts/notifications.coffee (hunk 2 above)
Comment (joel99) [reply scripts/notifications.coffee]: What action do you want? Bounce the current one? I thought overwrite would be a could default behavior.

File: scripts/notifications.coffee (hunk 4 above)
Comment (joel99) [reply scripts/notifications.coffee]: ah can I just not include anything, that's cool too, I'll do that

File: scripts/notifications.coffee [hunk 5]
```diff
@@ -39,9 +40,9 @@ GRAFANA = {
   tags: {}
 }
 
-PRESETS = ['buildgt', 'mediums', 'help', 'approve', 'abort']
+PRESETS = ['hackers', 'mediums', 'help', 'approve', 'abort']
 
-CONTROL_CHANNEL_NAME = '#hackgt5_announcements'
+CONTROL_CHANNEL_NAME = 'bot-spam'#'#hackgt5_announcements'
```
Comment (ehsanmasdar) [inline scripts/notifications.coffee]: Switch back to the real channel

**Review by ehsanmasdar — APPROVED** [review]

**Pull request merged**